packages/legacy-scripting-runner/test/integration-test.js

```diff
@@ -646,6 +646,28 @@ describe('Integration Test', () => {
       });
     });
 
+    it('KEY_pre_poll, this binding', () => {
+      const appDef = _.cloneDeep(appDefinition);
+      appDef.legacy.scriptingSource = appDef.legacy.scriptingSource.replace(
+        'movie_post_poll_underscore',
+        'movie_post_poll'
+      );
+      const _appDefWithAuth = withAuth(appDef, apiKeyAuth);
+      const _compiledApp = schemaTools.prepareApp(_appDefWithAuth);
+      const _app = createApp(_appDefWithAuth);
+
+      const input = createTestInput(
+        _compiledApp,
+        'triggers.movie.operation.perform'
+      );
+      input.bundle.authData = { api_key: 'secret' };
+      return _app(input).then(output => {
+        const movies = output.results;
+        movies.length.should.greaterThan(1);
```

Comment:
From my understanding of the issue, this is the line that fails before the fix, right? Because `movies` is `undefined`? It's awesome to hear that a one-line fix can resolve that! Nice work! 🥇

Reply:
The test would error when `this.getMovesUrl()` is invoked so that's before this line. The error message would be `this.getMoviesUrl is not a function`, something like [this](https://travis-ci.com/zapier/zapier-platform/builds/146512707#L451).